static/js/directives/repo-view/repo-panel-tags.js

```diff
@@ -67,6 +69,34 @@ angular.module('quay').directive('repoPanelTags', function () {
         });
       };
 
+      var matchCosignSignature = function(tag) {
+        // matches cosign style tags and returns the match
+
+        var cosignNamingPattern = new RegExp('^sha256-[0-9a-f]{64}\.sig$');
+        return tag.match(cosignNamingPattern);
```

Comment:
Not sure if all tags are stripped of white spaces. If you want you can strip the white spaces before matching with the Regex.

Reply:
@Sunandadadi where can a tag get whitespaces from?